sapi/cli/tests/ext_loading.phpt

```diff
@@ -7,6 +7,8 @@ if (!file_exists($extDir . '/opcache.so') && !file_exists($extDir . '/php_opcach
     die('skip Opcache shared object not found in extension_dir');
 }
 ?>
+--XFAIL--
+LSAN bug on Clang 14 marks this as leaking
```

Comment:
Please move this to skipif and only xfail if we're actually using asan.

Reply:
Done